src/meili/map_matcher.cc

```diff
@@ -664,12 +697,15 @@ std::vector<MatchResults> MapMatcher::OfflineMatch(const std::vector<Measurement
       const auto& this_stateid = original_state_ids[time];
       const auto& next_stateid =
           time + 1 < original_state_ids.size() ? original_state_ids[time + 1] : StateId();
+
+      midgard::PointLL first_projection = results[time].lnglat;
+      midgard::PointLL last_projection = results[time + 1].lnglat;
       const auto& interpolated_results =
```

Comment:
this shouldnt be a reference and if you do the move iterator you need to remove const as well